src/clr-angular/data/datagrid/providers/selection.ts

```diff
@@ -315,36 +316,71 @@ export class Selection<T = any> {
     if (this._selectionType !== SelectionType.Multi || !this._items.displayed) {
       return false;
     }
-    const displayedItems: T[] = this._items.displayed;
-    const nbDisplayed = this._items.displayed.length;
+    /** make sure to exclude the locked items from the list */
```

Comment:
nit: Check through the comments in the PR. We should probably not use the `/**` notation for comments as they are for TSDoc and would show up in generated docs. We are planning on providing this in the future so we should only use that notation if we want it to show up in the docs.